tests/load/locustfiles/locustfile.py

```diff
@@ -66,13 +108,25 @@ def __exit__(self, *args) -> None:
 
 
 class AutopushUser(FastHttpUser):
-    wait_time = between(30, 35)
-
     def __init__(self, environment) -> None:
         super().__init__(environment)
         self.uaid: str = ""
         self.channels: dict[str, str] = {}
         self.ws: Any = None
+        self.headers = {"TTL": "60", "Content-Encoding": "aes128gcm", "Topic": "aaaa"}
+        self.encrypted_data = base64.urlsafe_b64decode(
+            "TestData"
+            + "".join(
+                [
+                    random.choice(string.ascii_letters + string.digits)
+                    for i in range(0, random.randrange(1024, 4096, 2) - 8)
+                ]
+            )
+            + "=="
+        )
+
+    def wait_time(self):
+        return self.environment.autopush_wait_time(self)
```

Comment:
Something is not evaluating correctly here, so that i'm not able to execute the load tests. Error message:
<img width="1087" alt="image" src="https://github.com/mozilla-services/autopush-rs/assets/3422688/2e529bdc-fcd0-404e-b42e-20fb8b6a44e7">

I've kept the default input:
<img width="371" alt="image" src="https://github.com/mozilla-services/autopush-rs/assets/3422688/251e3c63-2048-4bbb-8ba5-62ffe9fe217d">

Reply:
This is resolved in https://github.com/mozilla-services/autopush-rs/pull/438